packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -50,64 +48,61 @@ export class OrgList implements vscode.Disposable {
     }
   }
 
-  public async getAuthInfoObjects() {
-    const authFilesArray = await AuthInfo.listAllAuthorizations().catch(
+  public async getOrgAuthorizations(): Promise<OrgAuthorization[] | null> {
+    const orgAuthorizations = await AuthInfo.listAllAuthorizations().catch(
       err => null
     );
 
-    if (authFilesArray === null || authFilesArray.length === 0) {
+    if (orgAuthorizations === null || orgAuthorizations.length === 0) {
       return null;
     }
-    const authInfoObjects: FileInfo[] = [];
-    for (const authFile of authFilesArray) {
-      try {
-        const filePath = path.join(
-          await ConfigFile.resolveRootFolder(true),
-          '.sfdx',
-          authFile.username + '.json'
-        );
-        const fileData = readFileSync(filePath, 'utf8');
-        authInfoObjects.push(JSON.parse(fileData));
-      } catch (e) {
-        console.log(e);
-      }
-    }
-    return authInfoObjects;
+
+    return orgAuthorizations;
```

Comment:
When I hover over `orgAuthorizations`, VSC's intellisense says it's of type "any"

maybe change line 52 to:
```
    const orgAuthorizations: OrgAuthorization[] = await AuthInfo.listAllAuthorizations().catch(
      err => null
    );
```

Reply:
I removed this catch block - don't love this silent failure - let me know how the new version looks to you

Reply:
looks good, thanks!